qa/SidechainTestFramework/account/httpCalls/transaction/createKeyRotationTransaction.py

```diff
@@ -4,21 +4,22 @@
 def http_create_key_rotation_transaction_evm(sidechainNode, key_type, key_index, new_key,
                                              signing_key_signature, master_key_signature, new_key_signature,
                                              nonce=None, gas_limit=300000, max_fee_per_gas=900000000,
-                                             max_priority_fee_per_gas=900000000, api_key=None):
+                                             max_priority_fee_per_gas=900000000, api_key=None, include_gas_info=True):
```

Comment:
Since we already have default values of gas inside the endpoint, I think we can remove the default value here in the python code and avoid to add the new **include_gas_info**

Reply:
Done
Added a case with gas_limit too low to check if gas limit is used in sdk